pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -301,21 +328,18 @@ func (r *ReconcileUserAccount) addFinalizer(userAcc *toolchainv1alpha1.UserAccou
 	return nil
 }
 
-// manageCleanUp deletes the identity, user and finalizer when the UserAccount is being deleted
-func (r *ReconcileUserAccount) manageCleanUp(userAcc *toolchainv1alpha1.UserAccount) error {
+// deleteUserAndIdentity deletes the identity, user and finalizer when the UserAccount is being deleted
+func (r *ReconcileUserAccount) deleteUserAndIdentity(userAcc *toolchainv1alpha1.UserAccount) (bool, error) {
+
 	if deleted, err := r.deleteIdentity(userAcc); err != nil || deleted {
```

Comment:
If the deletion is successful does that mean `deleted` == true ? If so, doesn't that mean you will return a result of `false` for a successful deletion?  I may be missing something here.

Reply:
@sbryzak yeah, i'm returning false because to set the status to disabled i'd expect that both the user/identity are already deleted (in other words the deletion should have failed due to "is not found" error). If i had to delete the user/identity then the status should be set to disabling. hope that makes sense!

Reply:
@alexeykazakov @sbryzak since we agreed to do this in separate reconcile loops i'm going to mark this as resolved - please re-open if there is something i missed.